docs/src/main/sphinx/release/release-401.md

```diff
@@ -0,0 +1,42 @@
+# Release 401 (19 Oct 2022)
+
+## General
+
+* Add support for using path-style access for all requests to S3 when using
+  fault-tolerant execution with exchange spooling. This can be enabled with the
+  `exchange.s3.path-style-access` configuration property. ({issue}`14655`)
+* Add output buffer utilization distribution to `EXPLAIN ANALYZE VERBOSE`. ({issue}`14596`)
+* Add operator blocked time distribution to `EXPLAIN ANALYZE VERBOSE`. ({issue}`14640`)
+
+## Delta Lake connector
+
+* Add support for writing to tables using [version 4 of the writer](https://docs.delta.io/latest/versioning.html#features-by-protocol-version).
+  This does not yet include support for [change data feeds](https://docs.delta.io/2.0.0/delta-change-data-feed.html)
+  or generated columns. ({issue}`14573 `)
+* Fix bug where executing a `COMMENT` or `ALTER TABLE ... ADD COLUMN` statement
+  would overwrite the reader and writer versions. ({issue}`14611`)
+* Fix failure when listing tables from the Glue metastore and one of the Glue
+  tables has no properties. ({issue}`14577`)
+
+## Hive connector
+
+* Fix failure when listing tables from the Glue metastore and one of the tables
+  has no properties. ({issue}`14577`)
+
+## Iceberg connector
+
+* Improve performance of the `remove_orphan_files` table procedure. ({issue}`13691`)
+* Fix query failure when analyzing a table containing a column with a
+  non-lowercase name. ({issue}`14583`)
+* Fix failure when listing tables from the Glue metastore and one of the Glue
+  tables has no properties. ({issue}`14577`)
+
+## Kafka connector
+
+* Add support for configuring the prefix for internal column names using the
+  `kafka.internal-column-prefix` catalog configuration property. The default
+  value is `_` to maintain current behavior. ({issue}`14224`)
```

Comment:
```suggestion
  value is `_`. ({issue}`14224`)
```

Reply:
I'm not sure this is an improvement, I'd rather provide the context that you don't need to change it and current behavior is unchanged.